api/envoy/config/response_map/v3/response_map.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
```

Comment:
The intention here is for this to be non-extension config (for source/common/response_map/*) and to have separate extension config for the response map filter (source/extensions/http/filters/response_map/...).

Is this idea correct or misguided?

Reply:
I looked harder at how the API is organized and I think I understand. My feeling now is that this should live under `api/envoy/extensions/common/reponse_map` since I intend for it to be shared, eventually, by both the HCM and ResponseMap filter. Does that make sense?